contracts/upgradeable_contracts/modules/fee_manager/OmnibridgeFeeManagerConnector.sol

```diff
@@ -77,7 +77,7 @@ abstract contract OmnibridgeFeeManagerConnector is Ownable {
                 } else {
                     IBurnableMintableERC677Token(_token).transfer(address(manager), fee);
```

Comment:
The token contract can be written as so it fails if the recipient of the funds is the fee manager. It could be done to avoid taking fees. 
There are two possible outcomes here:
1. In case of the Home-to-Foreign direction it will fail entire transaction. Let's leave the token owners to be responsible for the users claims.
2. In case of the Foreign-to-Home direction it will revert the mediator code execution. I believe we would like to minimize such reverts. So, it make sense to consider a special handler here to not revert the mediator operations if the transfer of fees fails. What do you think?

Since we allow setting of an alternative bridged token implementation for particular pair of tokens, similar is applicable for the `mint` operation -- the token code could forbid minting tokens for the fee manager.

Reply:
In any case, the malicious token contract can fake the behaviour of `transfer()`. It can even return `true` out of it, but do not make an actual transfer.
Moreover, since we have 0% for Foreign->Home direction on both bridges for now, the second change seems redundant. It will also violate the other comments from security auditors about `safeTransfer`/`safeMint` calls.
So, I would rather not to make such changes for now.